### PR TITLE
feat(ci): Prevent cancelling running jobs of a matrix

### PR DESCRIPTION
**Pull request by armenzg**
The default behaviour cancels running jobs if one of the jobs in a matrix fails.
This means that you would have to re-run all of them rather than just the one that failed.

This is a waste of resources and has the increased probability that a flaky test could make the other runs also fail, thus, requirying to re-run all again.

This change in combination to Github's new UI that allows re-running just one job (or the failed ones) makes it easier to get to a green run by just re-running what has failed.

### CODE REVIEW

File: .github/workflows/acceptance.yml [hunk 1]
```diff
@@ -19,6 +19,9 @@ jobs:
     runs-on: ubuntu-20.04
     timeout-minutes: 20
     strategy:
+      # This helps not having to run multiple jobs because one fails, thus, reducing resource usage
+      # and reducing the risk that one of many runs would turn red again (read: intermittent tests)
+      fail-fast: false
```
Comment (armenzg) [inline .github/workflows/acceptance.yml:24]: @billyvg I tried using `defaults` at the workflow level, however, that also requires defining `matrix` at the workflow level, thus, going back to this.

**Review by billyvg — COMMENTED** [review]
Should this be default on all ci workflows?

**Comment by armenzg** [comment]
> Should this be default on all ci workflows?

@billyvg it only makes sense when there's a matrix greater than one.

**Comment by billyvg** [comment]
> > Should this be default on all ci workflows?
> 
> @billyvg it only makes sense when there's a matrix greater than one.

True - we're missing at least the backend tests, not sure if there are others

**Comment by armenzg** [comment]
> > > Should this be default on all ci workflows?
> > 
> > 
> > @billyvg it only makes sense when there's a matrix greater than one.
> 
> True - we're missing at least the backend tests, not sure if there are others

How did I do that!? Thanks!

**Review by billyvg — APPROVED** [review]
Awesome!

Can we do this for getsentry as well?

**Comment by armenzg** [comment]
> Awesome!
> 
> Can we do this for getsentry as well?

That's a mythical repo I always forget about 😝

**Pull request merged**